sdk/core/Azure.Core/src/Pipeline/HttpPipeline.cs

```diff
@@ -131,7 +133,7 @@ public HttpMessage CreateMessage(RequestContext? context, ResponseClassifier? cl
         /// <returns>The <see cref="ValueTask"/> representing the asynchronous operation.</returns>
         public ValueTask SendAsync(HttpMessage message, CancellationToken cancellationToken)
         {
-            message.CancellationToken = cancellationToken;
+            message.SetCancellationToken(cancellationToken);
```

Comment:
why did you have to change this line of code? Is the property setter not backward compatible?

Reply:
Yes, the property was internally settable before:
https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/HttpMessage.cs#L68

Since we made it protected internal in ClientModel (https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/System.ClientModel/src/Message/PipelineMessage.cs#L88), we need a different way to set it internally in Azure.Core now.